docs/user_manual/projects.md

```diff
@@ -87,7 +88,10 @@ trackers to MISO.
 Files can be attached to a project and will show up in the Project Files section of the Edit Project page. For more
 information about attachments, see the [Attachments section](../attachments/).
 
+### Assigning Assays to Projects
 
+In the Edit Project page, there will be an `Assays` table (between `Project Files` and `Studies`). There, you may 
+assign assays to the selected project.
```

Comment:
* remove all backticks (inconsistent with the rest of the document)
* describe _how_ to add and remove assays
* add a link to the Assays of the user manual (inside the Requisitions page) for more information about assays